apis/v1alpha1/tlsroute_types.go

```diff
@@ -76,38 +78,36 @@ type TLSRouteRule struct {
 // given action.
 type TLSRouteMatch struct {
 	// SNIs defines a set of SNI names that should match against the
-	// SNI attribute of TLS CLientHello message in TLS handshake.
+	// SNI attribute of TLS ClientHello message in TLS handshake.
 	//
 	// SNI can be "precise" which is a domain name without the terminating
 	// dot of a network host (e.g. "foo.example.com") or "wildcard", which is
 	// a domain name prefixed with a single wildcard label (e.g. "*.example.com").
-	// The wildcard character '*' must appear by itself as the first DNS
-	// label and matches only a single label.
-	// You cannot have a wildcard label by itself (e.g. Host == "*").
+	// The wildcard character '*' must appear by itself as the first DNS label
+	// and matches only a single label. You cannot have a wildcard label by
+	// itself (e.g. Host == "*").
+	//
 	// Requests will be matched against the Host field in the following order:
 	//
-	// 1. If SNI is precise, the request matches this rule if
-	//    the SNI in ClientHello is equal to one of the defined SNIs.
-	// 2. If SNI is a wildcard, then the request matches this rule if
-	//    the SNI is to equal to the suffix
-	//    (removing the first label) of the wildcard rule.
+	// 1. If SNI is precise, the request matches this rule if the SNI in
+	//    ClientHello is equal to one of the defined SNIs.
+	// 2. If SNI is a wildcard, then the request matches this rule if the
+	//    SNI is to equal to the suffix (removing the first label) of the
+	//    wildcard rule.
+	// 3. If SNIs is unspecified, all requests associated to the gateway TLS
+	//    listener will match.
```

Comment:
```suggestion
	// 3. If SNIs is unspecified, all requests associated to the gateway TLS
	//    listener will match. This can be used to define a default backend for a TLS listener.
```